static/app/views/replays/detail/layout/index.tsx

```diff
@@ -81,7 +82,7 @@ function ReplayLayout({layout = LayoutKey.TOPBAR}: Props) {
 
   if (layout === LayoutKey.NO_VIDEO) {
     return (
-      <BodyContent>
+      <BodyContent style={{gridTemplateRows: hasNewTimeline ? '1fr auto' : 'auto 1fr'}}>
```

Comment:
👍 I think if styles get more complicated than this, we should refactor to have a separate styled component, but I think this is simple enough + temporary